### PR TITLE
Fix build for PHP 5.5 and earlier versions

### PR DESCRIPTION
**Pull request by derschatta**
the build was failing with due to issues with ICU. I updated the base build with PHPBrew and modified the other versions which depend on it.

While doing that I also fixed the sourcing of custom shell scripts as this was done on build and not on runtime.

### CODE REVIEW

File: php/php54/Dockerfile [hunk 1]
```diff
@@ -47,18 +46,4 @@ ENV PATH /root/.phpbrew/php/php-$PHPVERSION/bin:$PATH
...
-RUN apt-get update && apt-get install -y zsh
-RUN sh -c "$(curl https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)" --unattended
-RUN git clone https://github.com/romkatv/powerlevel10k ~/.oh-my-zsh/custom/themes/powerlevel10k
-RUN git clone https://github.com/zsh-users/zsh-autosuggestions ~/.oh-my-zsh/custom/plugins/zsh-autosuggestions
-RUN git clone https://github.com/zsh-users/zsh-completions ~/.oh-my-zsh/custom/plugins/zsh-completions
-RUN git clone https://github.com/zsh-users/zsh-syntax-highlighting ~/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
-RUN echo 'setopt +o nomatch' > ~/.zshrc
-RUN echo 'source ~/custom_shell/.zshrc' >> ~/.zshrc
-RUN cat ~/.bashrc >> ~/.zshrc
+CMD ["/bin/bash", "-c", "/root/.phpbrew/php/php-$PHPVERSION/sbin/php-fpm"]
```
Comment (codyfinegan) [inline php/php54/Dockerfile:49]: @derschatta Should this also be dropped? Was dropped in 53 & 55 I'm assuming in favor of the new entrypoint

Comment (derschatta) [reply php/php54/Dockerfile:49]: yes you are right. should be dropped

Comment (derschatta) [reply php/php54/Dockerfile]: done @codyfinegan

**Comment by derschatta** [comment]
@codyfinegan could you review/test this again if you find some time?

**Comment by codyfinegan** [comment]
@derschatta  Shall do

**Review by codyfinegan — APPROVED** [review]

**Comment by codyfinegan** [comment]
@derschatta Looks good, checked the 5.3, 5.4 & 5.5 images

**Pull request merged**